src/config.cpp

```diff
@@ -244,13 +245,17 @@ void cmdConfigReaperOptimal(Command* command) {
 		return;
 	}
 	const ReaperSetting settings[] = {
+		// Undock Media Explorer
 		{"reaper_explorer", "docked", 0, 0},
+		// Enable "Close window on escape key" option in Media Explorer menu
+		{"reaper_explorer", "autoplay", 0, 17},
```

Comment:
Unfortunately, I'm fairly sure this is going to break things. The fact that we need the value 17 here suggests this is one of those annoying config values that combines multiple settings into one using bit flags. That means that if someone has one of the other affected settings set to something other than the default, this change will clobber it.

My educated guess is that the flag we want is either 1 or 16 (1 << 4), since 17 is most likely a combination of those two bits.

You could try disabling this option again and checking the value of the autoplay key, but it's possible REAPER just removes it when all flags are set to the default. In that case, you'll need to find some other setting that also impacts this autoplay key (probably Media Explorer autoplay settings). After that, you can disable the "Close window on escape key" setting and see what the difference is. For example, say you find some other setting which changes that autoplay key from 17 to 25. Now you disable "Close window on escape key" and the value goes down to 9. Now, you know the bit flag you want is 16 (1 << 4).